tests/test_api_lib_policy.py

```diff
@@ -1313,6 +1314,39 @@ def test_01_sms_identifier(self):
 
         delete_policy("sms1")
 
+    def test_22_push_registration_url(self):
+        from privacyidea.lib.tokens.pushtoken import PUSH_ACTION
+        g.logged_in_user = {"username": "user1",
+                            "role": "user"}
+        builder = EnvironBuilder(method='POST',
+                                 data={'serial': "OATH123456"},
+                                 headers={})
+        env = builder.get_environ()
+        # Set the remote address so that we can filter for it
+        env["REMOTE_ADDR"] = "10.0.0.1"
+        g.client_ip = env["REMOTE_ADDR"]
+        req = Request(env)
+        req.User = User()
+        req.all_data = {
+            "type": "push"}
+        # In this case we have no reqistration_url. We will raise an exception
+        self.assertRaises(PolicyError, pushtoken_add_config, req, "init")
+
+        # Set a policy for the registration URL
+        set_policy(name="push_pol",
+                   scope=SCOPE.ENROLL,
+                   action="{0!s}=http://test,{1!s}=11".format(PUSH_ACTION.REGISTRATION_URL,
```

Comment:
This test fails for me because it cannot find ``REGISTRATION_URL``:
```
======================================================================
ERROR: test_22_push_registration_url (tests.test_api_lib_policy.PrePolicyDecoratorTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/fred/privacyidea/privacyidea/tests/test_api_lib_policy.py", line 1339, in test_22_push_registration_url
    action="{0!s}=http://test,{1!s}=11".format(PUSH_ACTION.REGISTRATION_URL,
AttributeError: type object 'PUSH_ACTION' has no attribute 'REGISTRATION_URL'
```

Reply:
You are right. The registration URL in the policies is gone, since it is part of the firebase config.
Will do.

Reply:
done in fd13ec74